images/ci.sh

```diff
@@ -15,4 +15,5 @@ curl -sfL https://updates.duplicati.com/beta/duplicati_2.0.7.1-1_all.deb -o /tmp
 apt -y install /tmp/duplicati.deb
 
 # Requirements for UI tests
-npx playwright install-deps
\ No newline at end of file
+npx playwright install-deps
+npx playwright install chromium
```

Comment:
Don't you also need ffmpeg, firefox and webkit here?

```
Downloading Chromium 117.0.5938.62 (playwright build v1080) from https://playwright.azureedge.net/builds/chromium/1080/chromium-linux.zip
Downloading FFMPEG playwright build v1009 from https://playwright.azureedge.net/builds/ffmpeg/1009/ffmpeg-linux.zip
Downloading Firefox 117.0 (playwright build v1424) from https://playwright.azureedge.net/builds/firefox/1424/firefox-debian-12.zip
Downloading Webkit 17.0 (playwright build v1908) from https://playwright.azureedge.net/builds/webkit/1908/webkit-debian-12.zip
```

Shows those are also being downloaded.

Reply:
I believe we only have tests for chromium running at the moment ([link](https://github.com/storj/storj/blob/9e85d28d3efd0ab588ca404a9a7e949e70c11557/testsuite/playwright-ui/playwright.config.ts#L54-L64)), but it can't hurt to install all the default browsers. Updated PR